platform/viewer/src/service-worker.js

```diff
@@ -10,7 +10,7 @@ workbox.core.clientsClaim();
 
 // Cache static assets that aren't precached
 workbox.routing.registerRoute(
-  /\.(?:js|css)$/,
+  /\.(?:js|css|json5)$/,
```

Comment:
Prevents a bug if json5 configuration files are used.  Not really hit right now in this branch, but generally useful to include.